core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -142,12 +142,15 @@ private ParameterizedPathMapping(String prefix, String pathPattern) {
                 skeletonJoiner.add(token);
                 continue;
             }
+            // The paramName should not include colons.
+            // TODO: Implement param options expressed like `{bar:type=int,range=[0,10]}`.
```

Comment:
Thanks a lot for adding `TODO` here! Maybe we should create an issue about this for future follow-up. Interested in creating one with some ideas? (You could add more examples, etc)

Reply:
Sure! I'll create the issue!

Reply:
Thank you, @sato9818! :bow:

Reply:
Created the new issue https://github.com/line/armeria/issues/5633 👍